ext/fetch/lib.rs

```diff
@@ -293,7 +292,7 @@ where
         match data {
           None => {
             // If no body is passed, we return a writer for streaming the body.
-            let (stream, tx) = MpscByteStream::new();
+            let (tx, stream) = tokio::sync::mpsc::channel(1);
```

Comment:
Are you sure having a buffer size of 1 won't cause any dead locks?

Reply:
It's currently set to 1 in MpscByteStream, and I used the same number for `ext/http` as well. I think that I'd like to include a buffer here, but it would more likely be at the byte level rather than at the packet level.

Reply:
Sounds reasonable 👍